wdl-lint/tests/lints/trailing-comma/source.wdl

```diff
@@ -0,0 +1,47 @@
+#@ except: DescriptionMissing, MatchingParameterMeta
+
+version 1.2
+
+task foo {
+    meta {
+        description: {
+            help: "test"
```

Comment:
Maybe add comments saying what is okay and what isn't? It was clear to me from the code alone.